docs/LogGuide.md

```diff
@@ -1122,6 +1140,25 @@ You cannot count on the time remaining to be precise.
 In rare cases, the time will already have counted down a tiny bit.
 This matters for cases such as ucob Nael phase doom debuffs.
 
+The 'stacks' value may also indicate other information about the buff. For example, Mudra will show different stack
+values for different combinations of Mudra. The only way to be sure that you are getting a "real" stack value is by
+cross-referencing with game data (such as via Xivapi) to see what the game says the max stack amount for that ability
+is (0 is the max for most abilities).
+
+#### Refreshes, Overwrites, and Deaths
+
+If a buff is refreshed early, you will get another 26-line. You will not get a 30-line indicating that the existing
+buff has been removed. The same is true for when stacks are consumed or added.
+
+Most debuffs allow one player to place the debuff on each target. For some, such as Trick Attack, only one can be on
+the enemy at a time. If a buff is overwritten, a 30-line will be generated for the status effect that got bumped.
+
+Thus, it is sufficient to track buffs using a combination of caster, target, and status effect ID. A refresh or stack
+change will have the same caster, target, and status effect, while an overwrite will generate a 30-line anyway.
+
+When a unit dies, you will get 30-lines for buffs that were removed by it dying, so there is no need to handle that
+specially in order to have accurate buff tracking.
+
```

Comment:
This is somewhat misleading. The StatusRemove events in ACT are fabricated by ACT, as well as some of the StatusAdd events. The source of truth is the StatusList event.

Reply:
True, though the solution for that would probably be to just add the necessary documentation to the StatusEffectList event. The log guide as it exists currently has nothing useful for 38 lines.